docs/source/supported/amarel.rst

```diff
@@ -0,0 +1,144 @@
+================
+Amarel (Rutgers)
+================
+
+Platform user guide
+===================
+
+https://sites.google.com/view/cluster-user-guide
+
+General description
+===================
+
+* Resource manager - ``SLURM``
+* Launch methods (per platform ID)
+
+  * ``rutgers.amarel`` - ``SRUN``
+
+* Configuration per node (per queue)
+
+  * ``main`` queue (388 nodes)
+
+    * 12-56 CPU cores
+    * 64-256 GB of memory
+
+  * ``gpu`` queue (38 nodes)
+
+    * 12-52 CPU cores
+    * 2-8 GPUs
+    * 64-1500 GB of memory
+
+  * ``mem`` queue (7 nodes)
+
+    * 28-56 CPU cores
+    * 512-1500 GB of memory
+
+* Other available queues
+
+  * ``nonpre``
+
+    * jobs will not be preempted by higher-priority or owner jobs
+
+  * ``graphical``
+
+    * specialized partition for jobs submitted by the OnDemand system
+
+  * ``cmain``
+
+    * "main" partition for the Amarel resources located in Camden
+
+.. note::
+
+   Amarel uses the ``--constraint`` option in ``SLURM`` to specify nodes
+   features (`SLURM constraint <https://slurm.schedmd.com/sbatch.html#OPT_constraint>`_).
+   RADICAL-Pilot allows to provide such features within a corresponding
+   configuration file. For example, if you want to select nodes with "skylake"
+   and "oarc" features (see the list of `Available compute hardware <https://sites.google.com/view/cluster-user-guide#h.kyrykrouyxxz>`_),
+   please follow the next steps:
+
+   .. code-block:: bash
+
+      mkdir -p ~/.radical/pilot/configs
+      cat > ~/.radical/pilot/configs/resource_rutgers.json <<EOF
+      {
+          "amarel": {
+              "system_architecture": {"options": ["skylake", "oarc"]}
+          }
+      }
+      EOF
+
+Setup execution environment
+===========================
+
+Python virtual environment
+--------------------------
+
+Create a **virtual environment with** ``venv``:
+
+.. code-block:: bash
+
+   export PYTHONNOUSERSITE=True
+   module load python
+   python3 -m venv ve.rp
+   source ve.rp/bin/activate
```

Comment:
I'm not sure about this part, @AymenFJA can you please check/test it?

Reply:
Thank you, Aymen! updated accordingly (and added other option using your explored results)